src/FSharp.MetadataFormat/FSharp.MetadataFormat.fsproj

```diff
@@ -119,7 +119,7 @@
     </ProjectReference>
   </ItemGroup>
   <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v4.0')">
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.0'">
```

Comment:
@forki 

If we could somehow figure a way to condense the conditionals by defining properties in *proj files:

```<paket_fwk_dotnet_45_to_463>the conditional</paket_fwk_dotnet_45_to_463>```

This would reduce the diffs / line lengths to make them closer to "human readable"

It would also save some time to msbuild for evaluating a single variable instead of long expression.